src/components/Banner/BannerContextManager.js

```diff
@@ -10,12 +10,23 @@ import React, {useState, useLayoutEffect} from 'react';
 import BannerContext from './BannerContext';
 
 let activeBanner = null;
+let snooze_start_date = null;
+
+const today = new Date();
+function addTimes(date, days) {
+  const time = new Date(date);
+  const newTime = time.setDate(time.getDate() + days);
+  return newTime;
+}
 
 // Example usage:
 // activeBanner = {
-//   storageId: 'dismiss_banner_blm',
+//   storageId: 'react_banner_blm',
 //   normalHeight: 60,
 //   smallHeight: 80,
+//   campaign_start_date: '4 Sep 2020',
+//   campaign_end_date: '28 Oct 2020',
+//   snooze_duration: 7,
```

Comment:
```suggestion
//   snooze_duration: 7, // days
```

Reply:
Or we could rename the setting to `snooze_days`!

Reply:
I like it! "Says what it does"